content/examples/constants.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "constants"
+date: 2023-06-25T14:50:00-06:00
+draft: false
+---
+
+Constants store data that cannot be modified, just like inmutable variables.
+
+The key differece are that constants are defined ad deploytime and
+- Defined with the keyworkd `const` and named in SCREAMING_SNAKE_CASE
+- Are only declared in the global scope.
+- Never defined at runtime, they are defined at deployment.
```

Comment:
As this example concerns not only Starknet contracts, I think we should use the more general term "compilation" (or "compile-time") instead of "deployment".